File: Do zero ao Hello World/Sprint - 2/ex06.py
# Faça um Programa que peça as quatro notas de 10 alunos, calcule e armazene num vetor a média de cada aluno, imprima o número de alunos com média maior ou igual a 7.0

notas = []
mediaGeral = []
cont = 0

for i in range(10):
    print(f'\nAluno {i+1}')
    notas.clear()
    media = 0
    for j in range(4):
        notas.append(float(input(f'{j + 1}ª nota: ')))
    media = sum(notas) / 4
    print(media)
    mediaGeral.append(media)
    if media >= 7:
        cont += 1

print(f'As médias são: {mediaGeral} e o total de média maior ou igual a 7 é {cont}')

File: Do zero ao Hello World/Sprint - 2/ex06.go
package main

import "fmt"



func main(){

	generalAvg := [] float32{}
	count := 0
	var nota1, nota2, nota3, nota4 float32

	for i := 0; i < 10; i++{

		fmt.Println("\nAluno ", i + 1)
		fmt.Print("1ª nota: ")
		fmt.Scanln(&nota1)
		fmt.Print("2ª nota: ")
		fmt.Scanln(&nota2)
		fmt.Print("3ª nota: ")
		fmt.Scanln(&nota3)
		fmt.Print("4ª nota: ")
		fmt.Scanln(&nota4)
		avg := (nota1 + nota2 + nota3 + nota4) / 4
		generalAvg = append(generalAvg, avg)
		if avg >= 7{
			count += 1
		}
		fmt.Println(avg)
	}
	fmt.Println("As médias das notas apresentadas foram: ", generalAvg, " e o total de notas maior ou igual a 7 é ", count)
}